pages/common/jekyll.md

```diff
@@ -0,0 +1,15 @@
+# Jekyll
+
+> Jekyll is a simple, blog-aware, static site generator.
```

Comment:
I'm not sure that we need to say "Jekyll" in both the title and description. Perhaps we could remove the `Jekyll is` from the beginning?

Reply:
Good idea, i'll remove the `Jekyll is`.